testVectors.json

```diff
@@ -22,7 +22,7 @@
       "testCases": [{
         "digest": "0x96f25d34d30a1bf4e280d6bb38b228e8993c8b28222d2b000000000000000000",
         "offset": 2,
-        "error": 103,
```

Comment:
changing these felt a lot like whack-a-mole. still not sure how I feel about golang fixture testing!!!

Reply:
ya, the JSON is a bit of a hack to allow reuse of the vectors across multiple implementations. Golang typically prefers test cases to be hardcoded in a `.go` file

Reply:
that makes sense